src/cmd/flux-jobs.py

```diff
@@ -627,7 +645,7 @@ def main():
         fmt = args.format
     else:
         fmt = (
-            "{id:>18} {username:<8.8} {name:<10.10} {status_abbrev:>6.6} "
+            "{id:>20} {username:<8.8} {name:<10.10} {status_abbrev:>6.6} "
```

Comment:
the increase in id width probably in wrong commit? (flux-jobs: add output fields related to job expiration )

Reply:
Hm, I don't even remember making that change... I will nix it.